heralding/capabilities/vnc.py

```diff
@@ -55,8 +56,15 @@ class Vnc(HandlerBase):
         writer.write(challenge)
 
         client_response = await reader.read(1024)
-        session.add_auth_attempt('des_challenge', response=binascii.hexlify(client_response).decode(),
-                                 challenge=binascii.hexlify(challenge).decode())
-
         writer.write(AUTH_FAILED)
+
+        # try to decrypt the hash
+        dkey = crack_hash(challenge, client_response)
```

Comment:
There is a character limit issue here, for example. When entering _dragonba_ as password, heralding will crack this as _dragonba**ll**_:
`2019-07-01 17:21:19,816 (heralding.misc.session) vnc authentication attempt from 127.0.0.1:48774. Auth mechanism: decrypted, session id d7d7f89d-7343-467f-b328-e62e239e4e4b Credentials: {"password": "dragonball"}`

I guess that it's because our VNC implementation is limited to 8 char passwords... One solutions could be to truncate all passwords to 8 chars while reading them into the list...

Reply:
I think it would be better to reconsider the passwords' top to include into the file only passwords up to 8 characters of length.

Reply:
That would make the wordlist useless for cracking passwords for other capabilities.